media-sound/ncmpcpp/ncmpcpp-0.8.2.recipe

```diff
@@ -106,3 +109,8 @@ INSTALL()
 
 	strip$secondaryArchSuffix $binDir/ncmpcpp
```

Comment:
You can't have both the debuginfo and the call to strip.
(If we add the debuginfo package then haikuporter will strip the runtime for us.)